ProjectCleanUp

DDemoNZ opened the pull request:
README
fix init_db
fix some classes 
and some other fix

DDemoNZ committed:
README file
And some fix

DDemoNZ committed:
fix readme

DDemoNZ committed:
fix readme

DDemoNZ committed:
fix readme

Sofasmile commented on README.md:
```diff
@@ -1 +1,71 @@
-"# Internet-ShopRepository" 
+<h1>"Internet-Shop Project"</h1>
+<br><hr>
```
```suggestion
<hr>
```

Sofasmile commented on README.md:
```diff
@@ -1 +1,71 @@
+It's simple realization of the web-application - internet shop.
+This shop has simple UI which done by CSS and JSP pages(HTML, JSPTL).
+Project has: authentic and authorization filter.
+Dao layer with tro implementation: Lists and JDBC.
```
```suggestion
Dao layer with two implementation: Lists and JDBC.
```

Sofasmile commented on README.md:
```diff
@@ -1 +1,71 @@
+Service layer with logic between Dao and Servlets.
+View layer with pages.
+<hr>
+
```
<br>

Sofasmile commented on README.md:
```diff
@@ -1 +1,71 @@
+
+# <a name="Start"></a>Start
+* Click "Inject" button. Or you can register two users and to do the next steps.
+* Change admin role_id in table user_roles from 1 to 2.
```
в якій таблиці мені потрібно змінювати цифри? і навіщо?

DDemoNZ replied:
Просто не получается инжектить юзера что бы у него была роль администратора сразу
Вот и пункты инжект, а потом поменять админу роль в юзер_ролс на 2
Может можно как-то это сделать проще?

DDemoNZ replied:
Можно добавить sql строку которая будет сама менять роль? И указать что эту строку запустить после кнопки инициализации?

Tarabarnik replied:
Можеш змінювати роль на стррінці allUsers для адміна, на приклад, або створити метод, який додає роль за її назвою типу addRoleByRoleName(Long UserId, String role)

Sofasmile commented on README.md:
```diff
@@ -1 +1,71 @@
+<br>
+
+# <a name="User"></a>For user
+* User can watch item list</h5>
```
User can:
- watch item list
- add and delete items into his bucket
- create order from bucket with items
- delete your orders

Sofasmile commented on README.md:
```diff
@@ -1 +1,71 @@
+<br>
+
+# <a name="Admin"></a>For administrator
+* Can add/delete(manage) items into item list</h5>
```
Admin can:
- ...

Sofasmile commented on README.md:
```diff
@@ -1 +1,71 @@
+
+# <a name="Developer"></a>For developer
+* Open this project in IDE as Maven project
+* Install and configure Tomcat
```
як я повинна налаштувати Tomcat і MySQL?

Sofasmile commented on README.md:
```diff
@@ -1 +1,71 @@
+* Open this project in IDE as Maven project
+* Install and configure Tomcat
+* Install and configure MySQL
+* Run script init_db.sql in MySQL
```
де взяти скріпт?

Sofasmile commented on README.md:
```diff
@@ -1 +1,71 @@
+* Install and configure Tomcat
+* Install and configure MySQL
+* Run script init_db.sql in MySQL
+* Configure log4j.properties if you want to check log
```
де знаходиться цей файл?

Sofasmile commented on README.md:
```diff
@@ -1 +1,71 @@
+*
+* It can be running with SQL or with Lists implementation
+* If you want change impl to list
+* Change "BucketDao", "UserDao", "ItemDao", "OrderDao" in Factory from "JDBC impl" to "simple impl"
```
можна додати приклад, як саме потрібно змінити

DDemoNZ replied:
А можно оставить в комменте нужный код и просто указать типа что то что есть нужно закомментить, а с коментария наоборот?
Или для такого комментарии плохо использовать?

Tarabarnik replied:
Коментарі потрібні, щоб пояснити логіку певного блоку коду або типу того. проект, в якому треба змінити код, щоб він працював - це неготовий або неправильно сформований проект

DDemoNZ committed:
check init_db.sql -> work
cleanUp style.css
add GenericService interface with CRUD methods
cleanUp JDBCimpl's classes, remove useless methods
cleanUp project
change logger to constant
fix README.md

Tarabarnik commented on src/main/java/mate/academy/internetshop/dao/jdbc/BucketDaoJdbcImpl.java:
```diff
@@ -67,8 +63,17 @@ public Bucket create(Bucket bucket) throws DataProcessingException {
 
-    @Override
-    public List<Item> getItemsFromBucket(Long bucketId) throws DataProcessingException {
+    private Bucket getBucketFromRS(ResultSet resultSet) throws SQLException,
```
```suggestion
    private Bucket getBucketFromResultSet(ResultSet resultSet) throws SQLException,
```

DDemoNZ committed:
fix 404 start page -> index mapped to "/"

DDemoNZ committed:
some fix

DDemoNZ committed:
some fix

DDemoNZ committed:
some fix
